internal/errors/observability.go

```diff
@@ -17,6 +17,7 @@
 // Package errors provides error types and function
 package errors
 
+// ErrCollectorNotFound represents a function to generate an error that the observability collector is not found.
 var ErrCollectorNotFound = func() error {
 	return New("observability.collector not found")
 }
```

Comment:
```suggestion
var (
	ErrCollectorNotFound = func() error {
		return New("observability.collector not found")
	}
)
```
Should we fix it?

Reply:
Thank you for your comment :bow:
I think we should fix.